packages/endomoat/src/index.js

```diff
@@ -0,0 +1,145 @@
+const ENDO_ROOT_POLICY = 'root'
+const ENDO_WRITE_POLICY = 'write'
+const ENDO_WILDCARD_POLICY = 'any'
+const LAVAMOAT_ROOT_PKG = '<root>'
+const LAVAMOAT_PKG_POLICY_ROOT = '$root$'
+
+const RSRC_POLICY_PKGS = 'packages'
+const RSRC_POLICY_BUILTINS = 'builtins'
+const RSRC_POLICY_GLOBALS = 'globals'
+
+const {isArray} = Array
+
+/**
+ * @typedef {typeof ENDO_ROOT_POLICY} RootPolicy
+ * @typedef {typeof ENDO_WRITE_POLICY} WritePolicy
+ */
+
+/**
+ * @typedef {import('@endo/compartment-mapper').Policy<RootPolicy>} LavaMoatEndoPolicy
+ */
+
+const DEFAULT_ATTENUATOR = '@lavamoat/endomoat/attenuator/default'
+const PROPERTY_ATTENUATOR = '@lavamoat/endomoat/attenuator/property'
+
+/**
+ * @param {Record<string, boolean>} item - A value in `ResourcePolicy`
+ */
+function toEndoRsrcPkgsPolicyBuiltins(item) {
+  /** @type {Exclude<import('@endo/compartment-mapper').PackagePolicy['builtins'], undefined>} */
+  const policyItem = {}
+  for (const [key, value] of Object.entries(item)) {
+    if (key.includes('.')) {
```

Comment:
I like this selective attenuator setup. 

Tip: you can use the default attenuator for both globals and modules, because these are two separate exports. (deliberately designed that way)